api/blueprints/drbCollection.py

```diff
@@ -252,3 +304,31 @@ def constructOPDSFeed(
     )
 
     return opdsFeed
+
+#Deleting the rows of collection_editions that were in the original collection
+def removeEditionsFromCollection(dbClient, collection):
+    dbClient.session.execute(COLLECTION_EDITIONS.delete().where(COLLECTION_EDITIONS.c.collection_id == collection.id))
+
+#Inserting rows of collection_editions based on editionIDs array
+def addEditionsToCollection(dbClient, collection, editionIDs):
+    dbClient.session.execute(COLLECTION_EDITIONS.insert().values([ \
+        {"collection_id": collection.id, "edition_id": eid} \
+        for eid in editionIDs \
+    ]))
+
+#Inserting rows of collection_editions based on workUUIDs array
+def addWorkEditionsToCollection(dbClient, collection, workUUIDs):
+    collectionWorks = dbClient.session.query(Work)\
+            .join(Work.editions)\
+            .filter(Work.uuid.in_(workUUIDs))\
+            .all()
+
+    for work in collectionWorks:
+        collectionEditions = dbClient.session.query(Edition)\
+            .filter(Edition.work_id == work.id)\
+            .order_by(Edition.date_created.asc())\
+            .all()
```

Comment:
Nice, this looks like the right logic here!  You *could* do a `.limit(1)` and then `.scalar()` instead of `.all()`, but no big deal really, just slighty less data sent over.

Reply:
I'll apply this logic and then I'll merge the change into main.